### ISSUE TITLE
Remove remnants of JSON-style and combine character setup

### ISSUE THREAD

**Issue by EvanKnowles** [dagwud/woodlands#56]
Evan
I feel like we're torn a bit with assembling characters - we have the enum for initial stats, the constructor of the concrete class for auto-spells, and the spell choice command for menu
I propose moving everything into the concrete class, given that we need it for groovy unique abilities

John
Very happy with that. The initial stats thing is a hangover from the initial json idea, and now it’s just clunky

**Comment by EvanKnowles** [comment]
Has this been partially fixed?

**Comment by dagwud** [comment]
Yeah, sort of. I did some cleanup working on the spells, but the whole initialstats mess is still there.

I quite like the idea that we can add more creatures/weapons through json, but for the character classes it feels very overkill
